clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/ImmutableChangeCapturePubSubMessage.java

```diff
@@ -51,7 +51,7 @@ public PubSubTopicPartition getTopicPartition() {
   }
 
   @Override
-  public VeniceChangeCoordinate getOffset() {
+  public VeniceChangeCoordinate getPosition() {
```

Comment:
Could I get you to take a look in the internal repo and make sure this isn't a used method?  I think changing this name might potentially break some folks.  In fact, I'm pretty sure we'll break a few things.

Reply:
Yeah I checked internal code and this is definitely a breaking change.   Seems like my previous PR is also gonna break some tests: https://github.com/linkedin/venice/pull/1559